gui/wxpython/vnet/vnet_data.py

```diff
@@ -1377,7 +1377,6 @@ def SetValue(self, value, line, col):
 
     def SetUTurns(self, value):
         """Checked if checeBox is checed"""
-        useUTurns = value
```

Comment:
You have an empty method, that won't work. The original intention migth have been something like:
```
self.useUTurns = value
```
It doesn't really matter that much, this part of code seems dead.